configuration/driver/icedrv_step.F90

```diff
@@ -698,6 +700,85 @@ subroutine step_dyn_wave (dt)
+         use icedrv_flux, only: rdg_conv, rdg_shear, dardg1dt, dardg2dt
+         use icedrv_flux, only: closing
+         use icedrv_init, only: tmask
+         use icedrv_state, only: trcrn, vsnon, aicen, vicen, trcr_base
+         use icedrv_state, only: aice0, trcr_depend, n_trcr_strata, nt_strata
+   
+         ! column package includes
+         use icepack_intfc, only: icepack_step_advection_scm
+   
+         real (kind=dbl_kind), intent(in) :: &
+            dt      ! time step
+   
+         ! local variables
+   
+         integer (kind=int_kind) :: &
+            i,            & ! horizontal indices
+            ntrcr          !
+   
+         character(len=*), parameter :: subname='(step_advection_scm)'
+   
+         !-----------------------------------------------------------------
+         ! query icepack values
+         !-----------------------------------------------------------------
+   
+            call icepack_query_tracer_sizes(ntrcr_out=ntrcr)
+            call icepack_warnings_flush(nu_diag)
+            if (icepack_warnings_aborted()) call icedrv_system_abort(string=subname, &
+                file=__FILE__,line= __LINE__)
+   
+         !-----------------------------------------------------------------
+         ! Ice advection
+         !-----------------------------------------------------------------
+            ! Currently we only do ridging for the SHEBA ocean data type (in step_dyn_ridge)
+            if (trim(ocn_data_type) == "SHEBA") then
+               ! Currently only uniform (and none) advection is implemented
+               if (trim(ice_advc_type) == "uniform") then
+      
+                  do i = 1, nx
+         
+                  if (tmask(i)) then
+         
+                     call icepack_step_advection_scm(dt=dt,           ncat=ncat,       &
+                                                   nilyr=nilyr,     nslyr=nslyr,     &
+                                                   trcrn=trcrn(i,1:ntrcr,:),         &
+                                                   trcr_depend=trcr_depend(1:ntrcr), &
+                                                   n_trcr_strata=n_trcr_strata(1:ntrcr),&
+                                                   nt_strata=nt_strata(1:ntrcr,:),   &
+                                                   trcr_base=trcr_base(1:ntrcr,:),   &
+                                                   aicen=aicen(i,:),                 &
+                                                   vicen=vicen(i,:),                 &
+                                                   vsnon=vsnon(i,:),                 &
+                                                   aice0=aice0(i),                   &
+                                                   closing=closing(i) )               
+         
+                  endif ! tmask
+         
+                  enddo ! i
+               elseif (trim(ice_advc_type) /= "none") then
```

Comment:
To make this section of the code more clear, I suggest 

`   elseif (trim(ice_advc_type) == "none") then `
`     ! do nothing:  no ice will be advected in, thus increasing the open water area through ridging`
`  else`
`     [abort] `

but it's okay like this.

Reply:
done